src/internal/clusterstate/v2.6.0/pfsdb.go

```diff
@@ -19,6 +19,72 @@ import (
 	"github.com/pachyderm/pachyderm/v2/src/pfs"
 )
 
+var schemaUpdate = `
```

Comment:
This whole section is for release that already went out.  Why is this in the diff?

Reply:
We don't want to keep around the functions that have been added here in the rest of our code base (I think because of dependency import cycles), so we moved them into the cluster state packages to ensure previous migrations still work.